components/collective-navbar/menu.js

```diff
@@ -0,0 +1,126 @@
+import { defineMessages } from 'react-intl';
+
+import hasFeature, { FEATURES } from '../../lib/allowed-features';
+import { hasSection, NAVBAR_CATEGORIES } from '../../lib/collective-sections';
+import i18nCollectivePageSection from '../../lib/i18n-collective-page-section';
+
+import { Sections } from '../collective-page/_constants';
+
+const titles = defineMessages({
+  CONTRIBUTE: {
+    id: 'Contribute.allWays',
+    defaultMessage: 'All ways to contribute',
+  },
+  TRANSACTIONS: {
+    id: 'SectionTransactions.Title',
+    defaultMessage: 'Transactions',
+  },
+  EXPENSES: {
+    id: 'section.expenses.title',
+    defaultMessage: 'Expenses',
+  },
+  UPDATES: {
+    id: 'updates',
+    defaultMessage: 'Updates',
+  },
+  CONVERSATIONS: {
+    id: 'conversations',
+    defaultMessage: 'Conversations',
+  },
+  CONTACT: {
```

Comment:
I think we decided that "Contact" should only be in the Actions menu since it's an action to take - @piamancini @Memo-Es?

Reply:
Yes, contact is actionable from the Collective NavBar

Reply:
It's indeed an action, but it's also a page: https://opencollective.com/babel/contact.

I'll make the change, we should also update the design:
![image](https://user-images.githubusercontent.com/1556356/101734084-0e1f5b80-3ac0-11eb-9c7f-808d4c146064.png)